roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -107,30 +107,27 @@ def get_var(self, *keys, **kwargs):
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
 
-        convert = kwargs.get("type")
+        convert = kwargs.get("convert", lambda x: x)
         try:
-            if convert is None:
-                pass
-            elif convert in ["str", str]:
-                return str(value)
-            elif convert in ["int", int]:
-                return int(value)
-            elif convert in ["float", float]:
-                return float(value)
-            elif convert in ["bool", bool]:
+            if convert is bool:
                 return str(value).lower() in ["true", "yes", "1"]
```

Comment:
Ansible [implements its own bool filter](https://github.com/ansible/ansible/blob/5f73bdc3bf3821c957cddbf81febfc13b3de9af9/lib/ansible/plugins/filter/core.py#L117-L125). It's not exactly a standard but... we can be consistent.